docs/02-app/01-building-your-application/06-configuring/02-eslint.mdx

```diff
@@ -27,8 +27,10 @@ yarn lint
 >
 > ? How would you like to configure ESLint?
 >
-> ❯ Base configuration + Core Web Vitals rule-set (recommended)
```

Comment:
<img width="920" alt="CleanShot 2023-06-08 at 14 14 42@2x" src="https://github.com/vercel/next.js/assets/9113740/4acaecf8-6f1f-4fa3-b31a-fb4952810197">

It seems this is the latest.